### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by zzx004** [LS4GAN/uvcgan2#21]
I'm not sure if there is a model collapse during training. Could you provide the changes in the loss function for a successful training?

**Comment by usert5432** [comment]
Hi @zzx004,

> Could you provide the changes in the loss function for a successful training?

Sure. We have uploaded our models to [zenodo](https://zenodo.org/records/7826901). Each model contains a file called `history.csv` that shows the change of training losses.
